styles/morse.module.scss

```diff
@@ -0,0 +1,26 @@
+@use './common/button' as common_buttons;
+
+.buttons_box {
+  display: flex;
+  flex-wrap: nowrap;
+  margin: 0 auto;
+  grid-gap: 2vmin;
+  justify-content: space-evenly;
+  align-items: center;
+
+  @media (orientation: landscape) and (max-height: 699px) {
+    flex-direction: column;
+  }
+
+  & > * {
+    @include common_buttons.button;
+    height: 25vmin;
+    width: 28vmin;
+    min-width: unset;
+    min-height: unset;
+
+    & > p {
+      font-size: max(65px, 400%);
+    }
```

Comment:
Add also rule for `& > h5` to improve labels on small screens. Either hide it, or use responsible (dynamic) font size